drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -8394,6 +8394,7 @@ void RenderingDeviceVulkan::_free_internal(RID p_id) {
 		b.allocation = index_buffer->allocation;
 		b.buffer = index_buffer->buffer;
 		b.size = index_buffer->size;
+		b.buffer_info = VkDescriptorBufferInfo();
```

Comment:
I had to add this to get rid of the `may be used uninitialized` error (https://github.com/godotengine/godot/runs/3529064438#step:8:1681) due to the changes to `List`.

Reply:
I don't what what this buffer info should be assigned to.  I think probably submit this as a small separate PR though, as it shouldn't have anything to do with list.

Reply:
Opened #52464